File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    private int calculate(String a, String b, String op){
        int ans = 0;
        if(op.equals("+")) ans = Integer.parseInt(a) + Integer.parseInt(b);
        else if(op.equals("-")) ans = Integer.parseInt(a) - Integer.parseInt(b);
        else if(op.equals("*")) ans = Integer.parseInt(a) * Integer.parseInt(b);
        return ans;
    }
    public int solution(String binomial) {
        String[] arr = binomial.split(" ");
        return calculate(arr[0], arr[2], arr[1]);
    }
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    binomial = binomial.split()
    if binomial[1] == '+':
        return int(binomial[0]) + int(binomial[2])
    elif binomial[1] == '-':
        return int(binomial[0]) - int(binomial[2])
    else:
        return int(binomial[0]) * int(binomial[2])
    return 0